src/main/java/tatracker/logic/parser/ParserUtil.java

```diff
@@ -54,7 +54,17 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
      */
     public static Name parseName(String name) throws ParseException {
         requireNonNull(name);
-        String trimmedName = name.trim();
+        String trimmedName = name.trim().toLowerCase();
+        StringBuilder nameBuilder = new StringBuilder();
+        String[] nameParts = trimmedName.split(" ");
+
+        for (int i = 0; i < nameParts.length; ++i) {
+            nameBuilder.append(Character.toUpperCase(nameParts[i].charAt(0)));
```

Comment:
With the first statement, the name Part might have 0 length, so this might not work as expected.

Reply:
Ah my bad there's a trim before so this shouldn't happen